docs/integrations/amazon-aws/lambda.md

```diff
@@ -286,16 +286,14 @@ Enter a parse expression to create an “account” field that maps to the alias
 ```yml
 Rule Name: AwsObservabilityLambdaCloudWatchLogsFER
 Applied at: Ingest Time
-Scope (Specific Data): _sourceHost=/aws/lambda/*
+Scope (Specific Data): account=*region* _sourceHost=/aws/lambda/*
```

Comment:
put a space before region
it should be account=* region=* _sourceHost=/aws/lambda/*

Reply:
done!